packages/@aws-cdk/aws-lambda-event-sources/lib/kinesis.ts

```diff
@@ -4,16 +4,24 @@ import * as cdk from '@aws-cdk/core';
 import { StreamEventSource, StreamEventSourceProps } from './stream';
 
 export interface KinesisEventSourceProps extends StreamEventSourceProps {
+  /**
+   * The time from which to start reading, in Unix time seconds.
+   *
+   * @default Required when startingPosition is AT_TIMESTAMP.
```

Comment:
This is not the correct use of default. The `@default` tag is meant to describe what happens if left blank -- should be `@default - no timestamp`. The information about when it's required is important, but put that in the regular documentation instead.

Reply:
Updated